Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -13,11 +13,31 @@ configuration:
   defaultvalue: ""
   type: 0
   required: true
-- display: API Token
+- display: Use Cb Response
```

Comment:
remove this param, check in the tests that if API Token is provided then CB response, if API Key and connector provided then use CB defense